src/deepsparse/benchmark/benchmark_model.py

```diff
@@ -298,12 +298,23 @@ def main():
     orig_model_path = args.model_path
     args.model_path = model_to_path(args.model_path)
 
+    if args.num_streams:
+        _LOGGER.info("num_streams set to {}".format(args.num_streams))
+    elif not args.num_streams and scenario not in "singlestream":
+        # If num_streams isn't defined, find a default
+        args.num_streams = max(1, int(args.num_cores / 2))
+        _LOGGER.info(
+            "num_streams default value chosen of {}. "
+            "This requires tuning and may be sub-optimal".format(args.num_streams)
+        )
```

Comment:
This looks good to land as-is but for the elastic scheduler, when num_streams isn't defined, I think it makes sense to pass in None, and then query the engine for its preferred num streams parameter and use that in the python code for the number of requests to put in for benchmarking. We'll do this in a follow-up.

Reply:
Makes sense.